Mage.Common/src/test/java/mage/view/ValidateViews.java

```diff
@@ -0,0 +1,184 @@
+package mage.view;
+
+import org.checkerframework.checker.units.qual.C;
```

Comment:
is this import even used? I don't recognize it

Reply:
Uhm that might be one of the PR when my IDE was adding imports as I typed, but without auto clean on.